app/main.py

```diff
@@ -1 +1,30 @@
-# write your code here
+import matplotlib.pyplot as plt
+import random
+
+
+def flip_coin(cases: int) -> dict:
+    template = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0,
+                6: 0, 7: 0, 8: 0, 9: 0, 10: 0}
```

Comment:
What if you have to flip coin 100 times? 1000? :) Rewrite this assign in pythonic way.